huaweicloud/services/ecs/resource_huaweicloud_compute_volume_attach.go

```diff
@@ -17,9 +17,12 @@ import (
 
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/helper/mutexkv"
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
 )
 
+var mutexKV = mutexkv.NewMutexKV()
```

Comment:
Pls set as a Global var.   volumePls set as a Global var.   volumeId may  be mounted by other resourceId may  be mounted by other resource

Reply:
done.